tutorials/evoked/30_eeg_erp.py

```diff
@@ -141,7 +146,7 @@
 # data, see :ref:`tut-filter-resample`. Here, we'll apply a simple high-pass
 # filter for illustration:
 
-raw.filter(l_freq=0.1, h_freq=None)
+_ = raw.filter(l_freq=0.1, h_freq=None)
```

Comment:
```suggestion
raw.filter(l_freq=0.1, h_freq=None)
```
showing the updated info with frequencies is for me relevant